hooks/use-api-data.ts

```diff
@@ -1,13 +1,16 @@
-import axios from "axios"
-import { useEffect, useState } from "react"
+import axios from 'axios'
+import { useEffect, useState } from 'react'
 
 export const useApiData = <T>(path: string, defaultValue: any): T => {
-  const [ data, setData ] = useState<T>(defaultValue)
+  const [data, setData] = useState<T>(defaultValue)
 
   useEffect(() => {
-    axios.get<T>(path).catch(err => err.response).then(response => {
-      setData(response.data)
-    })
+    axios
+      .get<T>(path)
+      .catch((err) => err.response)
+      .then((response) => {
+        setData(response.data)
+      })
   }, [])
```

Comment:
I get `missing dependency: 'path'`

Reply:
That's a little weird because path is a parameter of the function. `yarn tsc` / `yarn lint` is clear.